src/ExpressionEvaluator/Core/Test/ExpressionCompiler/ExpressionCompilerTestHelpers.cs

```diff
@@ -105,6 +105,23 @@ internal static class ExpressionCompilerTestHelpers
             out string errorMessage,
             out CompilationTestData testData)
         {
+            return CompileExpressionWithRetry(
```

Comment:
Are there many callers to the original method?  Maybe we should just delete it and move them to the new version?